### ISSUE TITLE
Write more tests

### ISSUE THREAD

**Issue by carakas** [forkcms/forkcms#2280]
## Type

- Enhancement

## Problem description

We have a rather low test coverage at the moment. Useful tests that can make us more confident that everything is working as it should be are more than welcome

**Comment by klnjmm** [comment]
> Useful tests that can make us more confident

Can you give priority on class to tests ? Or it doesn't matter ?

**Comment by carakas** [comment]
@klnjmm Maybe first making sure that we have complete functional tests for all the actions. That way we can be sure when we ship that we don't ship with broken actions or pages

**Comment by devdrops** [comment]
@carakas working on it :wink:

**Comment by devdrops** [comment]
Seems like I've stumbled on some issues:

1. I couldn't find any instructions to run the project on my local environment.
2. So, I've tried `docker-compose up`, as there is a `docker-compose.yml` file present at the project root.
3. Everything goes ok, until I hit `localhost:80` and it says that some dependencies are missing - even with the `composer install` command on the `Dockerfile`
4. Then, I've executed `docker-compose exec app bash` to enter the environment and then I've executed `composer install`, which solved _3_
5. Then I refreshed the browser but got a permission error for `/var/www/html/var/logs/install/debug.log`, which I've "fixed" by executing `chmod 777 /var/www/html/var/logs/install/debug.log` from inside the Docker environment.
6. Then I've found a new problem: _Your server doesn't meet the minimum requirements to run Fork CMS._ A few more permissions were required.

It seems that the Docker environment still needs some fixes. If it's a good idea, I'd be really happy to help with this issue :)
